### ISSUE TITLE
[BUG] [Linter] H006 false positive when width, height set via CSS class

### ISSUE THREAD

**Issue by OscarVanL** [djlint/djLint#821]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: e.g. ubuntu 20.04
 MacOS 14.4
 - Python Version (``python --version``)
3.9.17
 - djLint Version (``djlint --version``)
1.34.1
 - template language: e.g. mustache
jinja

## Issue

There is a false positive for H006: ` Img tag should have height and width attributes` when these are set via a CSS class.

## How To Reproduce

Style an image's size via a CSS tag:
```
<!DOCTYPE HTML>
<html lang="en">

<head>
<style type="text/css">
.logo-img {
        width: 161px;
        height: 41px;
    }
</style>
</head>

<body>
<img class="logo-img" src="{{ foo_logo }}"/>
</body>
</html>
```
The linter will fail with H006 incorrectly.

## Contents of .djlintrc/pyproject.toml [tool.djlint]

```
[tool.djlint]
ignore="T002,T003,H021,H029,H030,H031"
```

**Comment by monosans** [comment]
H006 checks for the width/height attributes on <img>, which browsers use to reserve space before CSS loads and thus avoid layout shift - that's why the rule exists even when CSS later sets the rendered size. djLint is a static template linter and doesn't parse stylesheets, so it can't know a class sets dimensions; treating any class as satisfying H006 would gut the rule.

If your project sizes images via CSS deliberately, disable the rule with ignore = "H006" or suppress specific occurrences with {# djlint:off H006 #}. Closing as working-as-designed.
